services/storage/src/simcore_service_storage/settings.py

```diff
@@ -26,6 +26,7 @@ class Settings(BaseCustomSettings, MixinLoggingSettings):
     )
 
     STORAGE_MONITORING_ENABLED: bool = False
+    STORAGE_PROFILING_MIDDLEWARE_ENABLED: bool = False
```

Comment:
Suggestion: Just STORAGE_PROFILING ?
that we use a middleware or anything else is not really needed right?

Reply:
Very good point. I have performed this change for all the env vars